module/VuFindConsole/src/VuFindConsole/Command/Util/BrowscapCommand.php

```diff
@@ -0,0 +1,193 @@
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Console
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFindConsole\Command\Util;
+
+use Laminas\Cache\Psr\SimpleCache\SimpleCacheDecorator;
+use Psr\Log\LogLevel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Console command: browscap
+ *
+ * @category VuFind
+ * @package  Console
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class BrowscapCommand extends Command
+{
+    /**
+     * The name of the command (the part after "public/index.php")
+     *
+     * @var string
+     */
+    protected static $defaultName = 'util/browscap';
+
+    /**
+     * Cache manager
+     *
+     * @var \VuFind\Cache\Manager
+     */
+    protected $cacheManager;
+
+    /**
+     * VuFind configuration
+     *
+     * @var array
+     */
+    protected $config;
+
+    /**
+     * Constructor
+     *
+     * @param \VuFind\Cache\Manager $cacheManager Cache manager
+     * @param array                 $config       Configuration
+     */
+    public function __construct(
+        \VuFind\Cache\Manager $cacheManager,
+        array $config
+    ) {
+        parent::__construct();
+        $this->cacheManager = $cacheManager;
+        $this->config = $config;
+    }
+
+    /**
+     * Configure the command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Browscap Cache Manager')
+            ->setHelp('Manages the browscap cache.')
+            ->addArgument(
+                'function',
+                InputArgument::REQUIRED,
+                'Function to execute. Currently the only supported function is: update'
+            )
+            ->addOption(
+                'file-type',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Browscap file type (default, lite or full)',
```

Comment:
You say default here, but you use normal below.

Reply:
Fixed. Now "standard" to align with browscap.org definition.